translations/monero-core_nl.ts

```diff
@@ -1104,330 +1068,511 @@ De naam van het oude cachebestand wordt gewijzigd, zodat het later kan worden he
+        <location filename="../pages/Settings.qml" line="764"/>
         <source>Warning: There&apos;s only %1 GB available on the device. Blockchain requires ~%2 GB of data.</source>
         <translation>Waarschuwing: er is slechts %1 GB beschikbaar op dit apparaat. Voor de blockchain is ~%2 GB opslagruimte nodig.</translation>
     </message>
     <message>
-        <location filename="../pages/Settings.qml" line="636"/>
+        <location filename="../pages/Settings.qml" line="766"/>
         <source>Note: There&apos;s %1 GB available on the device. Blockchain requires ~%2 GB of data.</source>
         <translation>Let op: er is slechts %1 GB beschikbaar op dit apparaat. Voor de blockchain is ~%2 GB opslagruimte nodig.</translation>
     </message>
     <message>
-        <location filename="../pages/Settings.qml" line="640"/>
+        <location filename="../pages/Settings.qml" line="770"/>
         <source>Note: lmdb folder not found. A new folder will be created.</source>
         <translation>LMDB-map niet gevonden. Er wordt een nieuwe map gemaakt.</translation>
     </message>
     <message>
-        <location filename="../pages/Settings.qml" line="572"/>
-        <location filename="../pages/Settings.qml" line="644"/>
+        <location filename="../pages/Settings.qml" line="703"/>
+        <location filename="../pages/Settings.qml" line="774"/>
         <source>Cancel</source>
         <translation>Annuleren</translation>
     </message>
     <message>
-        <location filename="../pages/Settings.qml" line="160"/>
+        <location filename="../pages/Settings.qml" line="167"/>
         <source>Successfully rescanned spent outputs.</source>
         <translation>Uitgegeven outputs zijn opnieuw opgezocht.</translation>
     </message>
     <message>
-        <location filename="../pages/Settings.qml" line="170"/>
+        <location filename="../pages/Settings.qml" line="182"/>
         <source>Change password</source>
         <translation>Wachtwoord wijzigen</translation>
     </message>
     <message>
-        <location filename="../pages/Settings.qml" line="201"/>
+        <location filename="../pages/Settings.qml" line="207"/>
+        <source>Daemon mode</source>
+        <translation>Daemon-modus</translation>
+    </message>
+    <message>
+        <location filename="../pages/Settings.qml" line="215"/>
         <source>Local Node</source>
         <translation>Lokale node</translation>
     </message>
     <message>
-        <location filename="../pages/Settings.qml" line="215"/>
+        <location filename="../pages/Settings.qml" line="226"/>
         <source>Remote Node</source>
         <translation>Externe node</translation>
     </message>
     <message>
-        <location filename="../pages/Settings.qml" line="233"/>
+        <location filename="../pages/Settings.qml" line="240"/>
+        <source>Bootstrap node</source>
+        <translation>Externe node</translation>
```

Comment:
Wellicht handig om deze wel `Bootstrap` te laten, gezien dit in de GUI een andere functionaliteit heeft dan puur het gebruik van een externe node.